gensrc/thrift/Normalization.thrift

```diff
@@ -0,0 +1,61 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+namespace java org.apache.doris.thrift
+
+include "Exprs.thrift"
+include "Types.thrift"
+include "Opcodes.thrift"
+include "Descriptors.thrift"
+include "Partitions.thrift"
+include "PlanNodes.thrift"
+
+struct TNormalizedOlapScanNode {
+  1: optional i64 table_id
+  2: optional i64 index_id
+  3: optional bool is_preaggregation
+  4: optional list<string> key_column_names
+  5: optional list<Types.TPrimitiveType> key_column_types
+  6: optional string rollup_name
+  7: optional string sort_column
+  8: optional list<string> select_columns
```

Comment:
there is no selected_partition_ids and remapped_slot_ids, plz check if we need them or not?

Reply:
I don't think remapped_slot_ids is needed.
The select_columns contains this information and ignore select column order